portfolio/src/main/webapp/passions.html

```diff
@@ -81,5 +79,45 @@ <h4 class="pass-header">Chess</h4>
         </div>
       </div>
     </section>
+    <!-- Slideshow Section -->
+    <section class="page-section" id="sldeshow">
+      <div class="container">
+        <div class="text-center">
+          <h2 class="section-heading text-uppercase">Travel gallery</h2>
+          <h3 class="section-subheading text-muted">Here is a brief list with some of the things I enjoy doing in my spare time</h3>
+        </div>
+        <div class="slideshow-container">
+          <div class="slides my-fade">
+            <div class="numbertext">1 / 3</div>
+            <img src="images/delta-danube.jpg" class="rounded">
+            <div class="text">Delta Danube</div>
+          </div>
+
+          <div class="slides my-fade">
+            <div class="numbertext">2 / 3</div>
+            <img src="images/fagaras.jpg" class="rounded">
+            <div class="text">Fagaras Moutains</div>
+          </div>
+
+          <div class="slides my-fade">
+            <div class="numbertext">3 / 3</div>
+            <img src="images/sinaia.jpg" class="rounded">
+            <div class="text">Sinaia</div>
+          </div>
+          <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+          <a class="next" onclick="plusSlides(1)">&#10095;</a>
+        </div>
+
+        <br>
+
+        <div style="text-align:center">
+          <span class="dot" onclick="currentSlide(1)"></span> 
+          <span class="dot" onclick="currentSlide(2)"></span> 
+          <span class="dot" onclick="currentSlide(3)"></span> 
+        </div>
+      </div>
+    </section>
   </body>
+  <script src="script.js"></script>
+  <script src="https://use.fontawesome.com/releases/v5.13.0/js/all.js" crossorigin="anonymous"></script>
```

Comment:
What's the license for fontawesome?

Reply:
From what I know, fontawsome has a model divided into 2 sections. On one hand, they offer a limited free to use widgets. On the other hand, if you want some premium icons, you need to pay a subscription fee. Do I still need a license, if they provide a CDN?